cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -2231,16 +2231,24 @@ if((NOT ARROW_SIMD_LEVEL STREQUAL "NONE") OR (NOT ARROW_RUNTIME_SIMD_LEVEL STREQ
 else()
   set(ARROW_USE_XSIMD FALSE)
 endif()
+
+set(ARROW_XSIMD_REQUIRED_VERSION "8.1.0")
+
 if(ARROW_USE_XSIMD)
-  set(xsimd_SOURCE "BUNDLED")
-  resolve_dependency(xsimd)
+  # Distros don't always provide up to date xsimd packages
+  set(xsimd_SOURCE "AUTO")
```

Comment:
Can we remove this because `xsimd_SOURCE` is `AUTO` by default?

Reply:
If we find old xsimd in system, we build xsimd as fallback.

Reply:
I added it because some CI jobs set it to `SYSTEM`, IIRC asan/ubsan and trivis CI failed as we don't install xsimd, and distro provided xsimd version is too low.

Reply:
OK. We should add `xsimd_SOURCE=BUNDLED` to CI jobs that have `ARROW_DEPENDENCY_SOURCE=SYSTEM` and old xsimd system package. For example, we did it in `ci/docker/ubuntu-18.04-cpp.dockerfile`.

Reply:
Thanks @kou. Will do.